pkgs/top-level/all-packages.nix

```diff
@@ -20727,8 +20727,6 @@ in
 
   bristol = callPackage ../applications/audio/bristol { };
 
-  bs1770gain = callPackage ../applications/audio/bs1770gain { };
```

Comment:
I would still prefer if we add a throw alias to not create any eval errors anywhere. The message should just be that the package got removed.

Reply:
Agreed, done :)